src/constantinople_lab_to_nwb/mah_2024/mah_2024_requirements.txt

```
@@ -0,0 +1 @@
+ndx-structured-behavior @ git+https://github.com/rly/ndx-structured-behavior.git@main
```

switched to forked version until PR is merged on original
```suggestion
ndx-structured-behavior @ git+https://github.com/catalystneuro/ndx-structured-behavior.git@make_task_program_and_schema_optional
```